aspnetcore/performance/caching/memory.md

```diff
@@ -25,7 +25,15 @@ The `IMemoryCache` cache will evict cache entries under memory pressure unless t
 
 The in-memory cache can store any object; the distributed cache interface is limited to `byte[]`.
 
-### Cache guidelines
+## System.Runtime.Caching/MemoryCache
+
+<xref:System.Runtime.Caching>/<xref:System.Runtime.Caching.MemoryCache> ([NuGet package](https://www.nuget.org/packages/System.Runtime.Caching/)) is available for use in .NET Standard 2.0 or later libraries and implementations of .NET, including apps that target ASP.NET Core 2.0 or later and .NET Framework 4.5 or later.
```

Comment:
This sentence reads funny to me. Could we say something like the following instead?

...can be used with:

* .NET Standard 2.0 or later
* any [.NET implementation](https://docs.microsoft.com/dotnet/standard/net-standard#net-implementation-support) targeting .NET Standard 2.0 or later

For example, ASP.NET Core 2.0 apps targeting .NET Core or .NET Framework 4.6.1 or later."